partiql-planner/src/main/resources/partiql_plan_internal.ion

```diff
@@ -335,7 +342,7 @@ rel::{
       paths: list::[path],
       _: [
         path::{
-          root: '.rex.op.var',
```

Comment:
Check with @alancai98 on this .. I know that using a rex.op makes it possible to insert an err node. In this case, I think you should be inserting an error node on the unresolved variable rather than a silent `var(-1,-1)`